### PR TITLE
Use prefix from config and implement dynamic command handling

### LINKED ISSUE #11: Add support for configurable prefix
the current prefix is `!prefix` as a POC, we should look into changing this to a config option  with a sensible default.

### PR DESCRIPTION
**Pull request by cjcanlas01**
- Use prefix from get-config
- Implement dynamic command handling by setting all command files to a Discord Collection
- Remove `!` from prefix variable on the command file

Closes: #11

### CODE REVIEW

File: src/commands/eightball.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  prefix: '!eightball',
+  prefix: 'eightball',
```
Comment (bradtaniguchi) [inline src/commands/eightball.js:2]: The `!help` command also needs to be updated

**edit** with the changes merged in master, the `!stats` command also needs to be updated

File: src/commands/bootstrap.js [hunk 2]
```diff
@@ -28,11 +31,36 @@ module.exports = function bootstrap({ client, config }) {
...
+  }
+
   client.on('message', (message) => {
-    for (let command of commands) {
-      if (message.content.startsWith(command.prefix)) {
-        command.command(message);
-        break;
+    if (message.content.startsWith(config.PREFIX)) {
+      // Get command after prefix
+      let commandArgument = message.content.split(' ')[1];
```
Comment (bradtaniguchi) [inline src/commands/bootstrap.js]: change to `const` instead of `let`

File: src/commands/bootstrap.js [hunk 3]
```diff
@@ -28,11 +31,36 @@ module.exports = function bootstrap({ client, config }) {
...
+
+      // Check if there are no commands sent
+      if (commandArgument == undefined) {
```
Comment (bradtaniguchi) [inline src/commands/bootstrap.js]: Change to `!commandArgument` instead

**Review by bradtaniguchi — CHANGES_REQUESTED** [review]

File: src/commands/eightball.js (hunk 1 above)
Comment (naomi-lgbt) [reply src/commands/eightball.js:2]: All that needs to change is the removal of the `!` for the prefix value, correct?

File: src/commands/bootstrap.js (hunk 2 above)
Comment (cjcanlas01) [reply src/commands/bootstrap.js]: Will do.

File: src/commands/eightball.js (hunk 1 above)
Comment (cjcanlas01) [reply src/commands/eightball.js:2]: Yes.

**Comment by cjcanlas01** [comment]
@bradtaniguchi 

Pull request updated.

**Review by bradtaniguchi — APPROVED** [review]
LGTM 👍 🚀 :shipit:

**Pull request merged**